pegasus/table_connector.go

```diff
@@ -469,39 +406,17 @@ func (p *pegasusTableConnector) MultiDel(ctx context.Context, hashKey []byte, so
 	return err
 }
 
-func (p *pegasusTableConnector) doTTL(ctx context.Context, hashKey []byte, sortKey []byte) (int, error) {
-	if err := validateHashKey(hashKey); err != nil {
-		return 0, err
-	}
-	if err := validateSortKey(sortKey); err != nil {
-		return 0, err
-	}
-
-	key := encodeHashKeySortKey(hashKey, sortKey)
-	gpid, part := p.getPartition(hashKey)
-
-	resp, err := part.TTL(ctx, gpid, key)
-	if err == nil {
-		err = base.NewRocksDBErrFromInt(resp.Error)
-		if err == base.NotFound {
-			return -2, nil
-		}
-	}
-	if err = p.handleReplicaError(err, gpid, part); err != nil {
-		return -2, err
-	}
-	return int(resp.GetTTLSeconds()), nil
-}
-
 // -2 means entry not found.
 func (p *pegasusTableConnector) TTL(ctx context.Context, hashKey []byte, sortKey []byte) (int, error) {
-	ttl, err := p.doTTL(ctx, hashKey, sortKey)
-	return ttl, WrapError(err, OpTTL)
+	res, err := p.runPartitionOp(ctx, hashKey, &op.TTL{HashKey: hashKey, SortKey: sortKey}, OpTTL)
```

Comment:
seem can directly `return p.runPartitionOp(ctx, hashKey, &op.TTL{HashKey: hashKey, SortKey: sortKey}, OpTTL)`
You have: 
https://github.com/XiaoMi/pegasus-go-client/blob/0bbc49562f57a437f93755100ea119a9d71bdb0c/pegasus/op/ttl.go#L51-L62